book/listings/todo/8/window/mod.rs

```diff
@@ -387,11 +387,13 @@ impl Window {
 
         // ANCHOR: setup_actions
         // Create action to create new collection and add to action group "win"
-        let action_new_list = gio::SimpleAction::new("new-collection", None);
-        action_new_list.connect_activate(clone!(@weak self as window => move |_, _| {
-            window.new_collection();
-        }));
-        self.add_action(&action_new_list);
+        let action_new_collection = gio::SimpleAction::new("new-collection", None);
```

Comment:
if you switch to ActionEntry::builder, you would no longer need the clone! usage here :)

Reply:
Could you give me an example how I would use that?
I don't get it from the docs alone

Reply:
Nevermind, found a couple of examples in other apps.